website/docs/r/pi_virtual_serial_number.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: pi_virtual_serial_number"
+description: |-
+  Manages an EXISTING virtual serial number in IBM Power
+---
+
+# ibm_pi_virtual_serial_number
+Get, update or delete an existing virtual serial number.
+
+## Example usage
+The following example enables you to create a shared processor pool placement group with a group policy of affinity:
+
+```terraform
+resource "ibm_pi_virtual_serial_number" "testacc_virtual_serial_number" {
+  pi_serial   = "<existing virtual serial number>"
+  pi_cloud_instance_id      = "<value of the cloud_instance_id>"
+  pi_description = "<desired description for virtual serial number>"
+}
+```
+
+**Note**
+* Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
+* If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
+  * `region` - `lon`
+  * `zone` - `lon04`
+  
+  Example usage:
+
+  ```terraform
+    provider "ibm" {
+      region    =   "lon"
+      zone      =   "lon04"
+    }
+  ```
+
+**Note**
+* This resource CANNOT create a virtual serial number unless assigning to a power instance using 'auto-assign'. Otherwise, it can only be used to manage an existing virtual serial number.
```

Comment:
We can now create with this resource.

Reply:
Updated wording. Please comment if there is still any issues.